tests/checks/mock/test_splunk_topology.py

```diff
@@ -0,0 +1,362 @@
+# stdlib
```

Comment:
I think there for the dispatch and results call there are two cases, either the response itself is an error response, or it is a succesfull response but the json cannot be parsed. The test you added falls in the cannot parse json category i suppose.